extensions/wikidata/tests/src/org/openrefine/wikidata/schema/WikibaseSchemaTest.java

```diff
@@ -158,6 +158,7 @@ public void testEvaluateRespectsFacets()
         List<ItemUpdate> expected = new ArrayList<>();
         ItemUpdate update1 = new ItemUpdateBuilder(qid1).addStatement(statement1).build();
         expected.add(update1);
+        System.out.println(expected);
```

Comment:
We can probably remove this `System.out.println` call.

Reply:
Sorry， I forget to delete this